funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -412,17 +412,22 @@ def list_endpoints():
 
 @app.command("delete")
 @name_arg
+@click.option(
+    "--force",
```

Comment:
default=False in the click options probably better than method default, as it is displayed to user on help

Reply:
Good call. I've adopted this suggestion in a fresh commit.